csi/moac/watcher.js

```diff
@@ -201,6 +201,51 @@ class Watcher extends EventEmitter {
     }
   }
 
+  // Fetches the latest object(s) from k8s and updates the cache; then it
+  // returns the k8s object(s) from the cache or null if it does not exist.
+  async getRawBypass (name) {
+    try {
+      const getObj = await this.getEp(name).get();
```

Comment:
would be good to narrow down the try-catch to just this line. Otherwise you catch all unexpected and programming errors not related to the GET request that normally should terminate the program.

Reply:
Yep done